src/requestor/request_utils.py

```diff
@@ -72,6 +73,30 @@ def get_redirect_url(action_id: str, data: dict) -> str:
     return final_redirect_url
 
 
+def get_credentials(creds_id: str) -> Tuple[str, str]:
+    # the config validation ensures the credentials exists
+    creds = config["CREDENTIALS"][creds_id]
+    if creds["type"] == "client_credentials":
+        # TODO we get a fresh access token every time. A potential improvement
+        # would be to cache/store the access tokens
+        logger.debug(
+            f"Attempting to get an access token from '{creds['config']['url']}'"
+        )
+        response = requests.post(
+            creds["config"]["url"],
```

Comment:
my personal preference is to keep like this when it's short enough, so that when i read the code i don't have to scroll up to check what the variable is. But i create a variable if it's too long and looks ugly after linting 🤷‍♀️